use-cases/README.md

```diff
@@ -0,0 +1,90 @@
+# Dapr use cases
+
+This document briefly describes scenarios that Dapr enables for your microservices. Each use case is accompanied with a link to learn more details.
+
+## Service invocation
+
+Dapr provides a uniform API that your microservice can use to find and reliably communicate with other microservices in your system. By using this API, your code can seamlessly take advantage of several built-in features:
+
+- HTTP and [gRPC](https://grpc.io) support
+- Robust retry logic
+- Intelligent error handling
+- Distributed tracing
+
+Visit the following documents to learn more:
+
+- [More details on service invocation](https://github.com/dapr/docs/blob/master/concepts/service-invocation/README.md)
+- [How to get started with service invocation](https://github.com/dapr/docs/tree/master/howto/invoke-and-discover-services)
+
+
+## State management
+
+Dapr provides an abstraction API that abstracts over [many different databases](https://github.com/dapr/docs/blob/master/howto/setup-state-store/supported-state-stores.md). Your microservice can leverage these APIs to utilize any supported database without finding or adding a third party SDK to your codebase.
```

Comment:
I think we should list Concurrency, Consistency, Retry Policies and Bulk Operations features here